src/ui/Forms/Main.cs

```diff
@@ -4469,6 +4469,7 @@ private void ResetSubtitle(bool forceVideoReload = false)
             _subtitle = new Subtitle(_subtitle.HistoryItems);
             _changeOriginalSubtitleHash = -1;
             _changeSubtitleHash = _subtitle.GetFastHashCode(GetCurrentEncoding().BodyName);
```

Comment:
@niksedk Shouldn't this be `_changeSubtitleHash  = -1` since we're resetting it?

Reply:
Yes, I think so.

Reply:
Fixed in 801a8a9e7bf32dea73a59bc48bcfb054dd7e897a